src/core/network.js

```diff
@@ -377,6 +377,10 @@ if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('FIREFOX || MOZCENTRAL')) {
       var networkManager = this._manager;
       var fullRequestXhrId = this._fullRequestId;
       var fullRequestXhr = networkManager.getRequestXhr(fullRequestXhrId);
+      if (networkManager.isHttp) {
```

Comment:
Hasn't this condition accidentally become inverted!?
As far as I understand, we want to prevent issues when the request is *not* `http` or `https`, and this condition actually does the opposite!
So, I'd assume that you mean:
```javascript
if (!networkManager.isHttp) {
  return false;
}
```

Also, let's move this code block up to just after the `var networkManager = this._manager;` line instead.